katsdpimager/frontend.py

```diff
@@ -444,6 +444,12 @@ def statistics(self, dataset, image_parameters, channel, **kwargs):
           Number of a pixels in the PSF patch, as a tuple (x, y).
         compressed_vis
           Number of compressed visibilities.
+        image_parameters
+          The :class:`ImageParameters` used for the channel.
+        grid_parameters
+          The :class:`GridParameters` used for the channel.
+        clean_parameters
```

Comment:
I don't see these being collected yet... Is that future work?

Reply:
It might be. I figured while I was doing the work to pass the other parameters I might as well pass all the things.